main/webapp/modules/core/scripts/views/data-table/column-header-ui.js

```diff
@@ -327,7 +327,7 @@ DataTableColumnHeaderUI.prototype._showSortingCriterion = function(criterion, ha
   elmts.blankErrorPositions.html(positionsHtml.join("")).sortable().disableSelection();
 
   var level = DialogSystem.showDialog(frame);
-  var dismiss = function() { DialogSystem.dismissUntil(level - 1); };
+  var dismiss = function() { DialogSystem.dismissLevel(level - 1); };
```

Comment:
Due to the order of the creation of the dialog, if use the dismissUntil the "waiting" window will get dismissed automatically when click the "sort" button. So dismiss the specific dialog to prevent the over kill.